src/rgw/rgw_op.cc

```diff
@@ -6127,7 +6128,7 @@ void RGWCompleteMultipart::execute(optional_yield y)
     ldpp_dout(this, 1) << "ERROR: publishing notification failed, with error: " << ret << dendl;
     // too late to rollback operation, hence op_ret is not set here
   }
-}
+} // RGWCompleteMultipart::execute
```

Comment:
i know it's cosmetic, but I do find this very helpful

Reply:
Whenever I have to search upward to figure out where I am and it's more than a couple of pages I try to remember to add one of these. RGW has long functions.